source/security/index.markdown

```diff
@@ -1,11 +1,81 @@
 
-Please do not make vulnerabilities public without notifying us and giving us at least 3 days to respond.
+This page is intended to provide information about how we handle security issues and information about security issues we have [handled in the past](#past-advisories).
 
-If you are going to write about Home Assistant's security, please get in touch, so we can make sure that all claims are correct.
-  
+## Reporting a vulnerability
+
+So, you have found a security vulnerability in Home Assistant? Please, be sure to **responsibly disclose** it to us by [reporting a vulnerability using GitHub's Security Advisory](https://github.com/home-assistant/core/security/advisories/new).
+
+**DO NOT MAKE A PUBLIC ISSUES FOR SECURITY VULNERABILITIES!**
+
+We are mostly interested in reports by actual Home Assistant users that are familiar with the platform, but all high quality contributions are welcome. Please do your best to describe a clear and realistic impact for your report.
+
+For the sake of the security of our users, please 🙏 do not make vulnerabilities public without notifying us and giving us at least 90 days to release a fixed version. We will do our best to respond to your report within 7 days and also to keep you informed of the progress of our efforts to resolve the issue, but understand that Home Assistant, like many open source projects, is relying heavily on volunteers that aren't full-time resources. We may not be able to respond as quickly as you would like due to other responsibilities.
+
+If you are going to write about Home Assistant’s security, please get in touch, so we can ensure that all claims are correct.
+
+### Non-qualifying vulnerabilities
+
+We will not accept reports of vulnerabilities of the following types:
+
+- Reports from automated tools or scanners.
+- Theoretical attacks without proof of exploitability.
+- Attacks that are the result of a third-party application or library (these should instead be reported to the library maintainers).
+- Social engineering.
+- Attacks that require the user to have access to the Home Assistant host system.
+- Attacks involving physical access to a user’s device, or involving a device or network that’s already seriously compromised (like, man-in-the-middle).
+- Attacks that require the user to install a malicious other software, like a third-party integration, add-on, or plugin.
+- Attacks that the user can only perform on themselves.
+- Privilege escalation attacks for logged in users. Home Assistant assumes every user is trusted and does not enforce user privileges. It assumes every logged in user has the same access as an owner account ([more information](/docs/authentication/#user-accounts)).
+
+### Supported versions
+
+We only accept reports against the latest stable & official versions of Home Assistant or any versions beyond that currently in development or beta test. The latest version can be found [here](https://github.com/home-assistant/core/releases).
```

Comment:
"that are currently in development"

`The latest stable version can be found on [GitHub](https://github.com/home-assistant/core/releases).`

(as below about `[here]` links)

Reply:
Adjusted. Except for the stable mention in the last sentence, as the page also provide pre-releases.